server/apiserver/apiserver.go

```diff
@@ -164,6 +164,59 @@ func (api *ApiServer) CallUpdatePixel(x int, y int, r int, g int, b int, userID
 	}
 }
 
+
+/*
+ * Insert the new user id to the userlist
+ */
+func (api *ApiServer) HTTPUpdateUserList(w http.ResponseWriter, req *http.Request) {
+	// Only for testing
+	user_id := req.URL.Query().Get("user_id")
+	api.CallUpdateUserList(user_id)
+}
+
+/*
+ * Insert the new user id to the userlist
+ */
+func (api *ApiServer) CallUpdateUserList(user_id string) []byte {
+	byt := []byte("")
+	_, ifErr := api.GetLastUserModification(user_id)
```

Comment:
Once consensus have the their PR merged with the new error, we'll differentiate between these two cases of user not existing and error

Reply:
Do you want to add some TODO or something to indicate that you are coming back in the code? If not that is fine too! I just know that I get forgetful.